site/content/docs/5.2/utilities/colors.md

```diff
@@ -14,19 +14,29 @@ Colorize text with color utilities. If you want to colorize links, you can use t
 {{< colors.inline >}}
 {{- range (index $.Site.Data "theme-colors") }}
 <p class="text-{{ .name }}{{ with .contrast_color }} bg-{{ . }}{{ end }}">.text-{{ .name }}</p>
+<p class="text-{{ .name }}-emphasis">.text-{{ .name }}-emphasis</p>
 {{- end -}}
 {{< /colors.inline >}}
 <p class="text-body">.text-body</p>
 <p class="text-muted">.text-muted</p>
+
+<p class="text-body-emphasis">.text-body-emphasis</p>
+<p class="text-body-secondary">.text-body-secondary</p>
+<p class="text-body-tertiary">.text-body-tertiary</p>
+
```

Comment:
We also have a .text-black` that is not listed here.
(see also https://github.com/twbs/bootstrap/pull/37538)

Reply:
Saving for that PR.